explcheck/testfiles/TL2024-issues.txt

```diff
@@ -387,7 +387,7 @@ kfupm-math-exam.cls s204,s206
 knowledge.sty s103,s204,s205,s206,w101,w202
 koma-script-source-doc.cls s204
 kvmap.sty s103,s204
-l3benchmark.sty s103,s204,s205,w302
+l3benchmark.sty s103,s204,s205,w202,w302
```

Comment:
Validate this change by downloading the up-to-date files `l3obsolete.txt` and `explcheck-obsolete.lua` locally and checking the file `l3benchmark.sty` to see if this is a false alarm or not.

Reply:
I checked that this change is due to `\sys_if_timer_exist:F` being added to `l3obsolete.txt` on 2025-03-27. Since `\sys_if_timer_exist:F` is used in historical version of the file `l3benchmark.sty`, this change seems to be as expected.

In order to have more visibility into the generation of the file `explcheck-obsolete.lua`, I added the date of generation and the latest obsolete entry into the generated file in commit a92709855dbb69cb6095dfbbccf1cdc7aec0ed0e from this PR.